examples/persistent-volume-provisioning/README.md

```diff
@@ -25,12 +25,13 @@ metadata:
 provisioner: kubernetes.io/aws-ebs
 parameters:
   type: io1
-  zone: us-east-1d
+  zones: us-east-1d, us-east-1c
   iopsPerGB: "10"
 ```
 
 * `type`: `io1`, `gp2`, `sc1`, `st1`. See AWS docs for details. Default: `gp2`.
-* `zone`: AWS zone. If not specified, a random zone from those where Kubernetes cluster has a node is chosen.
+* `zone`: AWS zone. If neither zone nor zones is specified, a random zone in the same region as controller-manager will be chosen. Note: zone and zones parameters must not be used at the same time.
```

Comment:
Can we re-spec the default as "a random zone chosen from the set of zones in which cluster VMs exist" ?